src/docker-utils.coffee

```diff
@@ -90,6 +91,7 @@ do ->
 							.on('id', resolve)
 							.on('error', reject)
 					.on 'error', reject
+				.timeout(DELTA_TOTAL_TIMEOUT)
```

Comment:
It would be nice if this could be a timeout based on the `res.pipe` not having any data flow for X period (along with a timeout for the data to start flowing in the first place)